extensions/wikia/PortableInfobox/services/PortableInfoboxDataService.class.php

```diff
@@ -129,7 +134,11 @@ private function getImagesFromOneNodeImageData( $nodeImageData ) {
 	public function save( NodeInfobox $raw ) {
 		if ( $raw ) {
 			$stored = $this->get();
-			$stored[] = [ 'data' => $raw->getRenderData(), 'sources' => $raw->getSource() ];
+			$stored[] = [
+				'parser_tag_version' => PortableInfoboxParserTagController::PARSER_TAG_VERSION,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "parser_tag_version" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)